### ISSUE TITLE
Operator contracts for authorization not showing in applications page (Ropsten)

### ISSUE THREAD

**Issue by sthompson22** [keep-network/keep-core#1912]
### Background

Last week we deployed  an updated token-dashboard against Ropsten, from master.  Last fully functional version of the token dashboard deployed  was  https://github.com/keep-network/keep-core/releases/tag/v1.2.4-rc which did not include the the overview  page.

Doing a fresh delegation with authorizer set to the operator address results in beacon and tBTC operator contracts not appearing for authorization in the applications page.

Reviewing an older delegation, that had already authorized beacon and  tBTC operator contracts for  the current release shows no history of authorization.

By reverting the token-dashboard to the `v1.2.4-rc` client version https://github.com/keep-network/keep-core/releases/tag/v1.2.4-rc the applications page was restored to normal behavior.  I could both authorize operator contracts for new delegations, and see history for older delegations.

tl;dr think we're looking for some changes between and https://github.com/keep-network/keep-core/releases/tag/v1.2.4-rc  and https://github.com/keep-network/keep-core/tree/sthompson22/token-dash/use-tbtc-v1.0.3-rc

### How the issue was surfaced

- Deploy updated token-dashboard to keep-test (from master)
- [Branch](https://github.com/keep-network/keep-core/tree/sthompson22/token-dash/use-tbtc-v1.0.3-rc) for deployment where issue was found.

#### Fresh delegation

1. Get a fresh token grant for account.
2. Login via Metamask with account to token-dashboard.
2. Setup a new delegation.  Beneficiary, Authorizer, Operator all set to the same account.  100k KEEP.
3. Go to applications page, empty.

#### Check older delegation

1. Login via metamask with account to token-dashboard.
2. Double check delegation history (should be a delegation that  was done before the token-dashboard from master was deployed
3. Go to applications page, empty.

### How to reproduce

1.  Get a grant
2. Do a delegation  setting operator and authorizer to the same account
3. Check the applications page

**Comment by r-czajkowski** [comment]
As I can see the `v1.2.4-rc` version doesn't contain changes against smart contracts. I mean the dapp is up to date with smart contracts on the `master` branch.

**Comment by nikgrego** [comment]
Not sure it's a bug, but.. One of the strange things i had today, but never had before - tBTC application page showed no info about bonding, after refreshing few times all back as it supposed to be. 

![Скриншот 2020-07-17 20 04 42](https://user-images.githubusercontent.com/66184067/87827711-1ca47680-c884-11ea-929b-154acb95ab71.png)
![Скриншот 2020-07-17 20 16 42](https://user-images.githubusercontent.com/66184067/87827713-1dd5a380-c884-11ea-929e-5704b1e6116e.png)

**Comment by r-czajkowski** [comment]
I've double-checked at the current https://dashboard.test.keep.network and everthing works as expected. Please see screenshots below.

**Delegations**
![obraz](https://user-images.githubusercontent.com/57687279/119484847-b4051080-bd56-11eb-829c-febe9ce22ce2.png)

**Authorizations Random Beacon**
![obraz](https://user-images.githubusercontent.com/57687279/119485040-e878cc80-bd56-11eb-80b3-c840e08f3daf.png)

**Authorizations tBTC**
![obraz](https://user-images.githubusercontent.com/57687279/119485136-03e3d780-bd57-11eb-84e5-b979b72d1524.png)
![obraz](https://user-images.githubusercontent.com/57687279/119485166-0e05d600-bd57-11eb-8da5-758560a7416c.png)
